src/lib/mongodb.ts

```diff
@@ -7,11 +7,7 @@ import { MongoClient } from 'mongodb'
 // Required to use global._mongoClientPromise
 declare global {
   /* eslint-disable-next-line */
-  namespace NodeJS {
-    interface Global {
-      _mongoClientPromise: Promise<typeof MongoClient>
-    }
-  }
+  var _mongoClientPromise: Promise<typeof MongoClient>
```

Comment:
thought: Oh is this the source of the mongo lint errors we saw?

Reply:
Correct. I paired with Suz on this and we came to the conclusion to just ignore the var. I did some digging and long story short, it needed to be declared a different way. We're on Typescript 4.7, and I found that this error was something that came up after 4.3, so I think it was just some peer dependency weirdness that caused this to only show recently.